<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPersImageCreate</class>
 <widget class="QDialog" name="DialogPersImageCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSource">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDest">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDest">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>输入文件路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSource">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>输出文件路径</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxFocus">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>左上角X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>宽度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>左上角Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>长度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>焦距</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxLeftX">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>59</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-90000</number>
   </property>
   <property name="maximum">
    <number>90000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxWidth">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>59</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>90000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxLeftY">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>59</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-10000</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxLenth">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>59</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>90000</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogPersImageCreate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogPersImageCreate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
